cms/templates/certificates.html

```diff
@@ -0,0 +1,86 @@
+<%inherit file="base.html" />
+<%def name="online_help_token()"><% return "certificates" %></%def>
+<%namespace name='static' file='static_content.html'/>
+<%! import json %>
+<%!
+  from contentstore import utils
+  from django.utils.translation import ugettext as _
+%>
+
+<%block name="title">${_("Course Certificates")}</%block>
+<%block name="bodyclass">is-signedin course view-certificates</%block>
+
+<%block name="header_extras">
+% for template_name in ["certificate-details", "certificate-editor", "signatory-editor", "signatory-details", "basic-modal", "modal-button", "list"]:
+  <script type="text/template" id="${template_name}-tpl">
+    <%static:include path="js/${template_name}.underscore" />
+  </script>
+% endfor
+</%block>
+
+<%block name="requirejs">
+  require(["js/certificates/factories/certificates_page_factory"], function(CertificatesPageFactory) {
+      CertificatesPageFactory(${json.dumps(certificates)}, "${certificate_url}", "${course_outline_url}");
+  });
+</%block>
+
+<%block name="content">
+  <div class="wrapper-mast wrapper">
+    <header class="mast has-actions has-subtitle">
+      <h1 class="page-header">
+        <small class="subtitle">${_("Settings")}</small>
+        <span class="sr">&gt; </span>${_("Certificates")}
+      </h1>
+    </header>
+  </div>
+
+  <div class="wrapper-content wrapper">
+    <section class="content">
+      <article class="content-primary" role="main">
+        <div class="wrapper-certificates certificates-list">
+          <h2 class="title">${_("Certificates")}</h2>
```

Comment:
Remove this title / h2, unless other pages in studio also keep this pattern, in which case add the "sr" class to hide it visually.